src/ios/GooglePlus.m

```diff
@@ -26,164 +26,131 @@ - (void)handleOpenURLWithAppSourceAndAnnotation:(NSNotification*)notification
 
     if ([possibleReversedClientId isEqualToString:self.getreversedClientId] && self.isSigningIn) {
         self.isSigningIn = NO;
-        [[GIDSignIn sharedInstance] handleURL:url];
+        [GIDSignIn.sharedInstance handleURL:url];
```

Comment:
see all breaking changes in v6.0.0 - https://developers.google.com/identity/sign-in/ios/release